bin/run-in-docker.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -e
+
+# Synopsis:
+# Analysis runner for run.sh in a docker container
+# Takes the same arguments as run.sh (EXCEPT THAT SOLUTION AND OUTPUT PATH ARE RELATIVE)
+# Builds the Dockerfile
+# Runs the docker image passing along the initial arguments
+
+# Arguments:
+# $1: exercise slug
+# $2: **RELATIVE** path to solution folder (with trailing slash)
+# $3: **RELATIVE** path to output directory (with trailing slash)
+
+# Output:
+# Writes the analyasis results to a analyasis.json file in the passed-in output directory.
+# The results are formatted according to the specifications at https://github.com/exercism/docs/blob/main/building/tooling/analyzers/interface.md
```

Comment:
This is probably copied from the analyzer? Could you update it to the representer? Feel free to copy from https://github.com/exercism/generic-representer/tree/main/bin